xray-16/Makefile

```diff
@@ -0,0 +1,37 @@
+PORTNAME=       xray-16
+DISTVERSION=    2188-november-2023-rc1
+CATEGORIES=     games
+
+MAINTAINER=     freebsd@sysctl.cz
+COMMENT=        game engine for stalker games
```

Comment:
Please, use the official project description.

Reply:
I added description